### DISCUSSION TITLE
Score multipliers based on difficulty change (Difficulty Adjust, DT, Relax, Autopilot etc.)

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#19744] [Ideas]
Basic system of multipliers is outdated, I think. Static modifiers unsuitable for creating a fair leaderboards. So...
We have:
- **performance points system**, that can be used for score calculation
        _for example:_ `modifier = Math.sqrt(current_max_pp / nomode_max_pp) * static_modifiers`
- **ingame map difficulty**, that also can be used for that
       _in this case we have to create big separate scoring system_
       _Something like_ `modifier = Math.sqrt((current_starrate * extra_starrate_calc_func(FL, HD, etc...)) / nomode_starrate) * static_modifiers`

With both opinions even Relax score can be calculated correctly. I think, we need new multiplier system like this **or even better** _(if you have a better idea for static modifiers replacement...)_

**Comment by peppy** [comment]
As I've mentioned elsewhere, multipliers should likely be understandable to a user, not 1.214238423x based on random ratios.

The direction I'd prefer to see is less reliance on mod multipliers, rather than more focus on complicating them.
